astropy/time/core.py

```diff
@@ -1729,23 +1729,12 @@ class TimeDelta(Time):
     info = TimeDeltaInfo()
 
     def __init__(self, val, val2=None, format=None, scale=None, copy=False):
-        if isinstance(val, timedelta) and not format:
-            format = 'datetime'
-
         if isinstance(val, TimeDelta):
             if scale is not None:
                 self._set_scale(scale)
         else:
             if format is None:
-                try:
```

Comment:
This stanza was pointless, since the test gets done by the formats anyway.

Reply:
I tried changing it here, but it didn't have any effect. Figured it was handled by formats. :sweat_smile: